src/og/segment/SegmentLonLatWgs84.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+import { Extent } from "../Extent.js";
+import { SegmentLonLat } from "./SegmentLonLat.js";
+
+export class SegmentLonLatWgs84 extends SegmentLonLat {
+    constructor(node, planet, tileZoom, extent) {
+        super(node, planet, tileZoom, extent);
+        this.isPole = false;
+    }
+
+    _assignTileYIndexes(extent) {
+        var lat = extent.northEast.lat;
+        this.tileY = Math.round((90.0 - lat) / (extent.northEast.lat - extent.southWest.lat));
+
+        if (lat > 0) {
```

Comment:
Also, don't like this, but understand the reason why it is here. Another mark to think about how we can generalize it..